rcldotnet/RCLdotnet.cs

```diff
@@ -1457,5 +1462,24 @@ internal static void WriteToMessageHandle(IRosMessage message, SafeHandle messag
                 }
             }
         }
+
+        internal static string GetStringFromNativeDelegate<T>(RCLdotnetDelegates.NativeRCLGetStringType nativeDelegate, T safeHandle) where T : SafeHandle
```

Comment:
This method doesn't need to be generic:

```suggestion
        internal static string GetStringFromNativeDelegate(RCLdotnetDelegates.NativeRCLGetStringType nativeDelegate, SafeHandle safeHandle)
```

Reply:
Oh of course 👍